acra/configuring-maintaining/key-storing/_index.md

```diff
@@ -4,6 +4,43 @@ bookCollapseSection: true
 weight: 1
 ---
 
-**TODO: Expand**
+# Key management and key storage
 
-# Key management and key storage in general 
\ No newline at end of file
+Key management with key storage is core component of Acra responsible for security guarantees. The whole cryptography 
+used in Acra depends on key storage. Without great key management, even strong and correctly used cryptography can't 
+guarantee safety for data.
+
+Key management is a set of approaches and processes related to key storing, rotation, distribution, protection, 
+compartmentalization. Acra provides tools and pushes the processes in key management in the secure way to cover all 
+aspects of keys security and help to reach compliance in data security.
+
+## Keys protection
+
+Every private key that stored and used by Acra encrypted with [Secure Cell]({{< ref "/themis/crypto-theory/cryptosystems/secure-cell.md" >}})
+that designed and responsible for secure storage. To encrypt all private keys Acra needs master key for symmetric cryptography
+under the hood of Secure Cell. This master key may be passed to Acra in two ways:
+* environment variable
+* KMS
+  Passing master key via environment variable as simple as possible. You can find example on our page
+  [Necessary preparations]({{< ref "/acra/configuring-maintaining/general-configuration/necessary_prep.md" >}}).
```

Comment:
I guess we don't need that indentation, otherwise this master key description may be rendered right next to "KMS", not in the next line as you probably expected

```suggestion

Passing master key via environment variable as simple as possible. You can find example on our page
[Necessary preparations]({{< ref "/acra/configuring-maintaining/general-configuration/necessary_prep.md" >}}).
```